src/js/controller/instream-adapter.js

```diff
@@ -215,6 +215,9 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         const { newstate } = event;
         if (newstate === STATE_PLAYING) {
             _controller.trigger(AD_PLAY, event);
+            if (_this.noResume && _model.get('autoPause').pauseAds) {
+                _this.pause();
+            }
```

Comment:
Isn't `noResume` supposed to be true with postrolls on the list time? I'm concerned about `noResume` remaining false after unpausing, making playback impossible. Also concerned that the playlist might not end properly if `noResume` is overloaded by the changes above.

Reply:
No, `noResume` is false on postrolls - we use `_beforeComplete` to check if it is a postroll: https://github.com/jwplayer/jwplayer/blob/master/src/js/controller/instream-adapter.js#L515